sphinxcontrib/spelling/checker.py

```diff
@@ -18,6 +22,10 @@ class SpellingChecker(object):
 
     def __init__(self, lang, suggest, word_list_filename,
                  tokenizer_lang='en_US', filters=None, context_line=False):
+        if not have_enchant:
+            raise RuntimeError(
+                'Cannot instantiate SpellingChecker '
+                'without PyEnchant installed')
```

Comment:
Here's how you can chain the import error to show more details about the exception that caused it in the first place:
```suggestion
        if enchant_import_error is not None:
            raise RuntimeError(
                'Cannot instantiate SpellingChecker '
                'without PyEnchant installed',
            ) from enchant_import_error
```
(together with https://github.com/sphinx-contrib/spelling/pull/59/files#r460555742)